applications/CompressiblePotentialFlowApplication/custom_elements/compressible_potential_flow_element.h

```diff
@@ -462,7 +463,7 @@ class CompressiblePotentialFlowElement : public Element
     {
         if(rValues.size() != 1) rValues.resize(1);
 
-        if (rVariable == PRESSURE)
+        if (rVariable == PRESSURE_COEFFICIENT)
```

Comment:
This is much better, but can you also correct the function names `ComputePressure` -> `ComputePressureCofficient` etc.